src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.cs

```diff
@@ -70,6 +70,16 @@ internal abstract partial class AbstractAsynchronousTaggerProvider<TTag> : Foreg
         protected virtual IEnumerable<Option<bool>> Options => SpecializedCollections.EmptyEnumerable<Option<bool>>();
         protected virtual IEnumerable<PerLanguageOption<bool>> PerLanguageOptions => SpecializedCollections.EmptyEnumerable<PerLanguageOption<bool>>();
 
+        /// <summary>
+        /// This controls what delay tagger will use to let editor know about newly inserted tags
+        /// </summary>
+        protected virtual TaggerDelay NewTagsNotificationDelay => TaggerDelay.NearImmediate;
+
+        /// <summary>
+        /// This controls what delay tagger will use to let editor know about just deleted tags.
+        /// </summary>
+        protected virtual TaggerDelay OldTagNotificationDelay => TaggerDelay.NearImmediate;
```

Comment:
OldTag is not the right name.  This should be something like "RemovedTagNotificationDelay" vs "AddedTagNotificationDelay".